provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -475,6 +475,10 @@
         "organizationName": {
           "description": "The organization's name.",
           "type": "string"
+        },
+        "githubTeamID": {
+            "description": "The GitHub ID of the team to mirror. This is the only required parameter when creating a GitHub team -- all other parameters are taken from GitHub directly. Must be in the same GitHub organization that the Pulumi org is backed by.",
+            "type": "number"
```

Comment:
This needs to also be listed as an input property, right?

The schema still marks the other input properties as required, do those need to be made optional now?

Reply:
> The schema still marks the other input properties as required, do those need to be made optional now?

Updated so `name` is now optional, org and type are still needed to construct relevant URLs.

Reply:
Do we validate the various combinations in `Check`?

Reply:
> Most of this provider's checks seem to no-op like this

:(

>  If it's worth doing while we're in here I can implement it.

I think that it's worth doing in this case b/c there are different combinations of valid inputs depending on the team type.